datahub/oauth/admin/utils.py

```diff
@@ -52,7 +52,11 @@ def get_sso_user_profile(access_token):
 def get_adviser_by_sso_user_profile(sso_user_profile):
     """Get adviser by sso user profile details."""
     try:
-        user = Advisor.objects.get(email=sso_user_profile['email'], is_staff=True, is_active=True)
+        user = Advisor.objects.get(
+            sso_email_user_id=sso_user_profile['sso_email_user_id'],
```

Comment:
[I believe it should be `email_user_id` in the data coming from SSO](https://github.com/uktrade/staff-sso/blob/3b23f1ea5965ab55aaa8d24f2b377605a918c7f0/sso/user/serializers.py#L27):

```suggestion
            sso_email_user_id=sso_user_profile['email_user_id'],
```

Reply:
I missed that, sorry!